conan/tools/microsoft/msbuild.py

```diff
@@ -10,7 +10,7 @@ def msbuild_verbosity_cmd_line_arg(conanfile):
 
 
 def msbuild_arch(arch):
-    return {'x86': 'x86',
+    return {'x86': 'Win32',
```

Comment:
Bug not manifested, I think, because the only place where this would have been used was defaulted:

[conan/tools/microsoft/layout.py](/conan-io/conan/blob/b6bcdc5f24bb9ab5e4ca8143a42637b1f010d471/conan/tools/microsoft/layout.py#L13):

```python
    if conanfile.settings.arch != "x86":
        arch = msbuild_arch(conanfile.settings.arch)
```